CHANGES.rst

```diff
@@ -8,6 +8,9 @@ assign_wcs
   uncaught exception when SIP approximation fails to reach desired
   accuracy. [#7036]
 
+- Relaxed FITS WCS SIP fitting parameters for the tweakreg step to make the
```

Comment:
This code change is in the tweakreg step, so it should go in change log section for tweakreg, not assign_wcs.